app/phpunit.xml.dist

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These files were added in March, when I started  the project. I remember that these files were required in order to  work with the bundle (it errored without them), so I added them.